moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -381,6 +386,13 @@ const ProjectView = () => {
     data?.moped_project?.[0]?.moped_proj_phases?.[0]?.moped_phase;
   const isProjectDeleted = data?.moped_project[0]?.is_deleted;
 
+  /**
+   * This function exists to enable closing the Map tab
+   */
+  const onCloseTab = useCallback(() => {
+    handleChange(null, 0);
+  }, [handleChange]);
```

Comment:
implementing caching broke the ability to close the map tab. not sure why exactly, but something to do with the `ProjectView` component only re-rendering because our `client` instance getting rebuilt 🤷